extensions/adobe/experience/campaign/experienceevent.example.click.json

```diff
@@ -0,0 +1,100 @@
+{
+  "xdm:timestamp": "2017-09-26T15:52:25+00:00",
+  "xdm:metrics": {
+    "https://ns.adobe.com/xdm/data/metrics/web/link-clicks": {
+      "xdm:value": 1
+    }
+  },
+  "xdm:environment": {
+    "xdm:type": "browser",
+    "xdm:browserDetails": {
+      "xdm:name": "Chrome"
+    },
+    "xdm:operatingSystem": "MAC OS"
+  },
+  "xdm:device": {
+    "xdm:typeID": "IMac",
+    "xdm:typeIDService": "https://ns.adobe.com/xdm/external/adobecampaign"
+  },
+  "https://ns.adobe.com/experience/campaign/message": {
+    "id": 10000,
+    "profile": {
+      "xdm:person": {
+        "xdm:birthMonth": 1,
+        "xdm:birthDay": 3,
+        "xdm:birthYear": 1996,
+        "xdm:gender": "female"
+      },
+      "xdm:workAddress": {
+        "@id": "https://ns.adobe.com/entities/address/123",
+        "xdm:primary": true,
+        "xdm:city": "San Jose",
+        "xdm:stateProvinceISO": "CA",
+        "xdm:postalCode": "95110",
+        "xdm:countryCode": "US"
+      },
+      "xdm:workEmail": {
+        "xdm:primary": true,
+        "xdm:address": "jsmith@xyzinc.com"
+      }
+    },
+    "variant": "English",
+    "seedMember": false,
+    "quarantine": false,
+    "proofMember": false,
+    "controlGroupMember": false,
+    "testMember": true,
+    "outboundIP": "10.20.30.40",
+    "offerProposed": [
```

Comment:
@saurabhere you updated the schema to take out the offerProposed property but the examples still have it.

Reply:
missed it.